core/integration/steps/parties_place_the_following_orders.go

```diff
@@ -437,8 +449,10 @@ func parseSubmitOrderTable(table *godog.Table) []RowWrapper {
 		"fb trailing",
 		"ra price trigger",
 		"ra trailing",
-		"so expires in",
-		"so expiry strategy",
+		"ra expires in",
```

Comment:
Not entirely sure why rise above/fall below require distinct fields here, if I'm honest, maybe it'll become clear as I get further down in this PR

Reply:
In the real world the user would be able to set each one independently so it felt right to do the same for the feature tests. Also because before it was only set once, it wasn't exercising all the code paths as only one side had an expiry set. Yes there is now a little more to type but it does mean better tests.

Reply:
Makes sense, all good